packages/keystone/errors.js

```diff
@@ -100,16 +100,16 @@ const GQLErrorCode = {
  * Error object, that can be thrown in a custom GraphQL mutation or query
  *
  * @typedef GQLError
- * @property {String} [mutation] - name of mutation where the error has been occured
- * @property {String} [query] - name of query where the error has been occured
- * @property {Array.<String>} [variable] - path to mutation or query argument, that is a subject of an error
+ * @property {String?} [mutation] - name of mutation where the error has been occured
+ * @property {String?} [query] - name of query where the error has been occured
+ * @property {Array.<String>?} [variable] - path to mutation or query argument, that is a subject of an error
```

Comment:
It's a wrong syntax for JSDoc
Optional params are in square brackets.
Take a look at https://jsdoc.app/tags-param.html
```suggestion
 * @property {String} [mutation] - name of mutation where the error has been occured
 * @property {String} [query] - name of query where the error has been occured
 * @property {Array.<String>} [variable] - path to mutation or query argument, that is a subject of an error
```

Reply:
IDE supports both types. But fixed.